k2/csrc/ragged_ops.h

```diff
@@ -70,7 +70,7 @@ void MinPerSublist(Ragged<T> &src, T default_value, Array1<T> *min_values) {
 template <typename T>
 void LogSumPerSublist(Ragged<T> &src, T default_value, Array1<T> *dst_values) {
   static_assert(std::is_same<float, T>::value ||
-                std::is_same<double, T>::value);
+                std::is_same<double, T>::value, "");
```

Comment:
It's strange that my compiler does not complain this without "", but anyway, thanks! (Just thinking of that we have defined `K2_STATIC_ASSERT` in log.h, maybe we can use that.)

Reply:
Fixed.